contracts/axelar-operators/src/contract.rs

```diff
@@ -82,17 +104,17 @@ impl AxelarOperatorsInterface for AxelarOperators {
         contract: Address,
         func: Symbol,
         args: Vec<Val>,
-    ) -> Val {
+    ) -> Result<Val, OperatorError> {
         operator.require_auth();
 
         let key = DataKey::Operators(operator.clone());
 
         if !env.storage().persistent().has(&key) {
-            panic_with_error!(env, Error::NotAnOperator);
+            Err(OperatorError::NotAnOperator)?
```

Comment:
Just wondering, what is the best practice to use `?` ? From the Soroban examples, it uses `return Err( );` statement instead of `?` for error handling. i.e: https://github.com/stellar/soroban-examples/blob/e595983e7e7c30c2b745f0cdde1096319bb7c90c/account/src/lib.rs#L152

Reply:
yeah, a bit weird to use `?` in this manner. Create an `ensure` macro in the axelar-soroban-std library, something like:
```
#[macro_export]
macro_rules! ensure {
    ($cond:expr, $e:expr $(,)?) => {
        if !$cond {
            return Err($e);
        }
    };
}

# usage
ensure!(env.storage().persistent().has(&key), OperatorError::NotAnOperator);
```

Reply:
It probably compiles down to the same thing, but since `?` technically de-sugars down to a match statement I guess it's more appropriate to use `return` directly. No real reason for it, just a habit I've picked up.